presto-main/src/test/java/com/facebook/presto/operator/aggregation/differentialentropy/TestUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.differentialentropy;
+
+import org.testng.annotations.Test;
+
+import java.util.Random;
+
+import static com.facebook.presto.operator.aggregation.differentialentropy.EntropyUtils.calculateEntropyFromSamples;
+import static org.testng.Assert.assertEquals;
+
+public class TestUtils
+{
+    @Test
+    public void getEntropyFromSamplesUniform()
+    {
+        Random random = new Random(13);
+        double[] samples = new double[10000000];
+        double sigma = 0.5;
+        for (int i = 0; i < samples.length; ++i) {
```

Comment:
++i -> i++

Reply:
C++ roots :-)

Reply:
Changed it in all places found in the code.